Kovaleva.HomeWork-6/Program.cs

```diff
@@ -1,12 +1,42 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace Kovaleva.HomeWork_6
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Card card1 = new Card(long.Parse("1234 4567 3456 3212".Replace(" ", "")), 2311);
+            Card card2 = new Card(long.Parse("3323 5647 8900 6765".Replace(" ", "")), 1315, "", 1500);
+
+            ATM atm = new ATM(new List<Card>() { card1, card2 });
+
+            atm.CurrentCard = card1;
+            Console.WriteLine($"Введите pin карты {card1.CardNumber}") ;
+            int pin1 = int.Parse(Console.ReadLine());
+            if (atm.CheckPin(pin1))
+            {
+                Console.WriteLine("Какую сумму добавить?");
+                decimal sum = decimal.Parse(Console.ReadLine());
+                atm.operation = atm.AddSum;
+                atm.operation(sum);
+                atm.PrintBalance();
+            }
+
+            atm.CurrentCard = card2;
```

Comment:
try to reuse the same code for adding and withdrawing amount with help of delegates

*tip: operation looks more like a variable/parameter than a field of the class here